src/canvas.rs

```diff
@@ -80,7 +81,13 @@ impl DrawingBackend for CanvasBackend {
     type ErrorType = CanvasError;
 
     fn get_size(&self) -> (u32, u32) {
-        (self.canvas.width(), self.canvas.height())
+        let style = window().unwrap()
+          .get_computed_style(&self.canvas)
+          .unwrap().unwrap();
+        let width = style.get_property_value("width").unwrap().replace("px", "");
+        let height = style.get_property_value("height").unwrap().replace("px", "");
```

Comment:
Here we can handle it better if the CSS uses a different unit, like `pt` or `rem`.
Also, error handlings can be improved.

Please let me know if there is an interest in merging the pull request, then I will improve it further. Otherwise, this code is good enough for my use case at the moment.